account_financial_risk/views/res_partner_view.xml

```diff
@@ -20,54 +20,68 @@
                                        attrs="{'readonly': [('risk_allow_edit', '=', False)]}"/>
                                 <button name="open_risk_pivot_info" type="object" class="oe_link"
                                         context="{'open_risk_field': 'risk_invoice_draft'}">
-                                    <field name="risk_invoice_draft" nolabel="1"/>
+                                    <field name="risk_invoice_draft" nolabel="1" widget='monetary' options="{'currency_field': 'risk_currency_id'}"/>
```

Comment:
I think this is not necessary because it is already established in the field definition

Reply:
ok for the options, but the widget is good to add, right?

Reply:
Ok. Forget my comment.